src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -69,6 +70,16 @@ def list(self):
             osd['stats'] = {}
             osd['stats_history'] = {}
             osd_spec = str(osd_id)
+            metadata = str(mgr._ceph_get_metadata('osd', osd_spec))  # pylint: disable = W0212
```

Comment:
```suggestion
            metadata = str(mgr.get_metadata('osd', osd_spec))
```

Reply:
Both methods have the same output, however using mgr.get_metadata was giving an unusual 'Key Error' in the test_cephfs.py, so I went on using the _ceph_get_metadata method.

Reply:
Then we should investigate the cause of the failure and adapt the test accordingly.
We should call the public counterpart of the protected method.